locales/en-US/strings.properties

```diff
@@ -60,17 +60,17 @@ time_label_minute={number}m
 time_label_hour={number}h
 time_label_day={number}d
 
-# LOCALIZATION NOTE (settings_pane_*): This is shown in the Settings Pane modal
-# dialog.
+# LOCALIZATION NOTE (settings_pane_*): This is shown in the Settings Pane sidebar.
 settings_pane_button_label=Customize your New Tab page
 settings_pane_header=New Tab Preferences
-settings_pane_body=Show these items on my new tab page.
+settings_pane_body=Choose what you see when you open a new tab.
```

Comment:
I just confirmed with @aaronrbenson that we do indeed want it to be all lower case because it isn't really a proper name. merging!